packages/infection-monkey/PKGBUILD

```diff
@@ -0,0 +1,66 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=infection-monkey
+_pkgname=monkey
+pkgver=v1.8.2.r2596.g2bc0f739
+pkgrel=1
+pkgdesc='Automated security testing tool for networks.'
+arch=('any')
+groups=('blackarch' 'blackarch-networking' 'blackarch-automation')
+url='https://github.com/guardicore/monkey'
+depends=('python' 'python-cryptography' 'python-cffi'
+         'python-ecdsa' 'pyinstaller' 'impacket'
+	 'python-ipaddress' 'python-netifaces' 'python-odict'
```

Comment:
![image](https://user-images.githubusercontent.com/16578570/126604377-0a4ec6de-2a8b-4bfa-8486-195746612126.png)

I suggest you display boundary spaces in your text editor/IDE because you must be mixing spaces and tabs here. Eg. for vscode 
![image](https://user-images.githubusercontent.com/16578570/126604548-ce453332-39c1-41ad-a0a2-e9e0148ab1d7.png)